Dockerfile

```diff
@@ -81,8 +77,7 @@ RUN --mount=from=nemo-src,target=/tmp/nemo cd /tmp/nemo && pip install ".[all]"
 
 # Check install
 RUN python -c "import nemo.collections.nlp as nemo_nlp" && \
-    python -c "import nemo.collections.tts as nemo_tts" && \
-    python -c "import nemo_text_processing.text_normalization as text_normalization"
```

Comment:
Does dockerfile not need to install nemo_text_processing? Or do we need to add it to more of our requirements.txt files?

Reply:
nemo_text_processing is part of requirements_tts.txt